iOS_SDK/OneSignalSDK/Source/OneSignalLocation.m

```diff
@@ -218,8 +250,10 @@ + (void)internalGetLocation:(bool)prompt {
 
 - (void)locationManager:(id)manager didUpdateLocations:(NSArray *)locations {
```

Comment:
Would this method still fire if the user has accepted notifications but has disabled location in the device settings?

Reply:
Good question

Reply:
Yes! Discussed on the call 😁